src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Order/TreeItemOrderPropertyProvider.cs

```diff
@@ -48,13 +58,14 @@ private void ComputeIndices()
             // _displayOrderMap only.
             foreach (var item in OrderedItems)
             {
-                var includeParts = item.EvaluatedInclude.Split(CommonConstants.PathSeparators, StringSplitOptions.RemoveEmptyEntries);
+                var includeParts = GetStringParts(item.EvaluatedInclude);
                 foreach (var part in includeParts)
                 {
-                    var rootedPath = duplicateFiles.Contains(part) ? _project.MakeRooted(item.EvaluatedInclude) : null;
-                    if (rootedPath != null && !_rootedOrderMap.ContainsKey(rootedPath))
+                    // EvaluatedInclude is a unique relative path for a file.
+                    var duplicatePart = duplicateFiles.Contains(part) ? item.EvaluatedInclude : null;
```

Comment:
This isn't a duplicatedPart - but rather the path or rootedOrRelativePath of the item.

Reply:
In the previous code, that is what it meant. It doesn't mean that now. 

"duplicatePart" in this case is, for example, like "opt.fs" or "src" in "visualfsharp/src/opt.fs".  Though, we are only checking for duplicate files. In the future, we may check for duplicate folders, but there are some problems with that which are out of scope.

Reply:
You are assigning `duplicatedPart` `Item.EvaluatedInclude` which isn't any of those things.

Reply:
I see. I did not explain that well and I was incorrect. ```duplicatePart``` is not any of those and not a good name for this.